### PR TITLE
show list orders

### PR DESCRIPTION
**Pull request by ghost**
## Description
show list orders
## Reference Requirements
- Task ID: TI-1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

```
mysql> EXPLAIN select count(*) as aggregate from `orders` \G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: orders
   partitions: NULL
         type: index
possible_keys: NULL
          key: orders_user_id_foreign
      key_len: 4
          ref: NULL
         rows: 35
     filtered: 100.00
        Extra: Using index
1 row in set, 1 warning (0.00 sec)

ERROR: 
No query specified
```
```
mysql> EXPLAIN select * from `orders` limit 10 offset 0 \G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: orders
   partitions: NULL
         type: ALL
possible_keys: NULL
          key: NULL
      key_len: NULL
          ref: NULL
         rows: 35
     filtered: 100.00
        Extra: NULL
1 row in set, 1 warning (0.00 sec)

ERROR: 
No query specified
```
```
mysql> EXPLAIN select * from `users` where `users`.`id` in ('3', '5', '9', '18', '23', '30', '31') and `users`.`deleted_at` is null \G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: users
   partitions: NULL
         type: range
possible_keys: PRIMARY
          key: PRIMARY
      key_len: 4
          ref: NULL
         rows: 7
     filtered: 10.00
        Extra: Using where
1 row in set, 1 warning (0.00 sec)

ERROR: 
No query specified
```

### CODE REVIEW

File: app/Http/Controllers/Admin/OrdersController.php [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 use Illuminate\Http\Request;
 use App\Http\Controllers\Controller;
+use App\Order;
+use App\User;
```
Comment (at-tuanphan) [inline app/Http/Controllers/Admin/OrdersController.php]: User không dùng thì remove đi em

File: resources/views/admin/order/index.blade.php [hunk 2]
```diff
@@ -37,63 +36,37 @@
...
-                    <a href=""><i class="fa fa-info"></i></a>
-                  </td>
-                </tr>
+                  @foreach ($orders as $order)
+                  <tr>
+                    <td>{{ $order->id }}</td>
+                    <td>{{ $order->user->name }}</td>
+                    <td>{{ $order->user->email }}</td>
+                    <td>{{ $order->total }} &dollar;</td>
+                      @if ($order->status == 1)
```
Comment (at-tuanphan) [inline resources/views/admin/order/index.blade.php]: không hardcode em: 1 và 2 không có ý nghĩa, define ở model rồi gọi ra

**Review by ghost — COMMENTED** [review]
please check my comments

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**